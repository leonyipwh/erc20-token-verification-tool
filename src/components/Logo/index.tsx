import LogoSvg from '../../images/osl-logo.svg?component';
import Box from '@mui/material/Box';

const Logo = () => {
  return (
    <Box>
      <LogoSvg fill="white" />
    </Box>
  );
};

export default Logo;
