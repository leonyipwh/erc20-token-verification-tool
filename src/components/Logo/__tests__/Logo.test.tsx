import Logo from '../index';
import { render } from '@testing-library/react';

describe('<Logo>', () => {
  it('should render', () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
