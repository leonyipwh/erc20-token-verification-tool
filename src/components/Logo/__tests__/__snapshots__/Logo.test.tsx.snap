// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo> should render 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <svg-mock
      fill="white"
    />
  </div>
</DocumentFragment>
`;
